backend/app/api/endpoints/adapter/subscription_market.py

```
@@ -0,0 +1,153 @@
+        skip=skip,
+        limit=limit,
+    )
+    return RentalSubscriptionsListResponse(total=total, items=items)
+
+
+@router.get(
+    "/subscriptions/{subscription_id}/rental-count", response_model=RentalCountResponse
+)
+def get_rental_count(
+    subscription_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)